services/QuillLMS/app/services/clever_integration/district_teacher_integration.rb

```diff
@@ -43,5 +44,12 @@ def run
         user: teacher
       )
     end
+
+    private def auto_verify_teacher
+      return if !SchoolsUsers.exists?(user: teacher) || teacher.teacher_verification_status == TeacherInfo::APPROVED || teacher.admin_approval_status == AdminInfo::APPROVED
```

Comment:
I feel like these methods are violating law of demeter since they're reaching into teacher which then reaches into teacher_info.  They're also long to read and I think could benefit from a helper:

teacher.admin_info_approved?  or  teacher.teacher_verified? or whatever makes sense.